examples/model_selection/plot_cost_sensitive_learning.py

```diff
@@ -660,14 +660,17 @@ def business_metric(y_true, y_pred, amount):
 #
 # The class :class:`~sklearn.model_selection.FixedThresholdClassifier` allows us to
 # manually set the decision threshold. At prediction time, it behave as the previous
-# tuned model but no search is performed during the fitting process.
+# tuned model but no search is performed during the fitting process. Note that here
+# we use :class:`~sklearn.frozen.FrozenEstimator` to wrap the predictive model to
+# avoid any refitting.
 #
 # Here, we will reuse the decision threshold found in the previous section to create a
 # new model and check that it gives the same results.
```

Comment:
```suggestion
# Here, we simply reuse the decision threshold found in the previous section to create
# a new model that implements the same decision function. We wrap the underlying
# estimator with the `FrozenEstimator` to avoid retraining it when we call `fit`.
# We then check that we obtain the same results as previously.
```